circle.yml

```diff
@@ -40,7 +40,7 @@ deployment:
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
   bintray:
-    tag: /[0-9]+(\.[0-9]+){2}/
+    tag: /[0-9]+(\.[0-9]+){2}(-alpha|-beta)?/
```

Comment:
I think those brackets make those into possible character sets instead of string literals, I'm somewhat confirmed on https://regex101.com/.  Let me know if you think this is incorrect.